tests/calculations/configs/config_janus.yaml

```diff
@@ -1,4 +1,3 @@
 properties:
   - "energy"
-out: "results.xyz"
```

Comment:
Does this not work, or do you just not want to test it anymore?

Reply:
I removed it just to test something. But I think it should stay removed, cause the way the code is written at the moment it's always gonna use the default name for the output (which is what we want cause it's easier to handle for us and the user should not interact with the file directly anyways, so we know the output is always a xyz)

Reply:
So this isn't actually used at the moment anyway?

If you want to ensure the output is an xyz, you can also specify the format, which would also override any file extensions, but maybe that could be more confusing.